### PR TITLE
chore(DB/quest_template_locale): Align column names to quest_template table.

### PR DESCRIPTION
**Pull request by heyitsbench**
## Changes Proposed:
- Renames the columns in `quest_template_locale` to line up with those in `quest_template`

## Issues Addressed:

- Closes unreported issue.

## SOURCE:

WowPacketParser as well as the `quest_template` table itself.

## Tests Performed:

- Untested.

## How to Test the Changes:

1. Make sure quest localizations still work.

## Known Issues and TODO List:

- [X] ~~Kinda want to rename the variables and such as well, but since the original commit that changed the column names in `quest_template` didn't, I don't know if I need to. 🤷‍♀️~~ Renamed the variables/functions as well.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by heyitsbench** [comment]
Goes without saying, but this would bork any open PRs that rely on old column names, such as #14210.

**Review by acidmanifesto — APPROVED** [review]
unique change, my issue will be the re-run ability of the sql, which i am not certain can. would recomment merge before "Squash" other wise no reason to not approve this.

**Comment by heyitsbench** [comment]
> unique change, my issue will be the re-run ability of the sql, which i am not certain can.

Table changing queries generally can't be re-run, seen in another PR that that's just the nature of them and it's nothing to worry about.

> would recomment merge before "Squash" other wise no reason to not approve this.

Fine by me. 👍

**Pull request closed without merging**
